feat: clarify how to configure a REST connection (#7185)
camunda/camunda-docs: docs/components/modeler/desktop-modeler/troubleshooting.md

Ensure your local orchestration cluster is running. If you don't have one installed, consider [Camunda 8 Run](/self-managed/quickstart/developer-quickstart/c8run.md), a lightweight all-in-one distribution.

> ## I need a REST connection for some features

Reviewer: ```suggestion
## How to configure a REST connection for Desktop Modeler
```
More goal oriented and to better align with our voice tone in our style guide

You try out [task testing](./task-testing.md) and Desktop Modeler tells you "Configure a REST connection to a Camunda 8 cluster."

> Some features of Desktop Modeler, such as [task testing](./task-testing.md), require a REST connection to a Camunda 8 cluster. Most orchestration clusters support both gRPC and REST connections and depending on the provided URL the according protocol will be used. Ensure you use the REST URL in your deployment configuration:

Reviewer: ```suggestion
Some features of Desktop Modeler, such as task testing, require a REST connection to a Camunda 8 cluster. Most orchestration clusters support both gRPC and REST connections and depending on the provided URL the according protocol will be used. Ensure you use the REST URL in your deployment configuration:
```

Already linked above, not needed again here

> - For Camunda 8 SaaS clusters create an [API client](../../console/manage-clusters/manage-api-clients.md) and use the `Camunda REST API`,

Author: i've added a link in the previous section ("Orchestration clusters from version 8.6 support connections with gRPC or the newer [Orchestration Cluster REST API](../../../apis-tools/orchestration-cluster-api-rest/orchestration-cluster-api-rest-overview.md) and depending on the provided URL the according client will be used.")

> - if your using c8run you should use the value of `Orchestration Cluster API`

Reviewer 2: ```suggestion
- if you are using [Camunda 8 Run](../../../self-managed/quickstart/developer-quickstart/c8run/) you should use the value of `Orchestration Cluster API`
```
